components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -4,3 +4,6 @@ export const stripERC20TokenImageURL = (url?: string) =>
 export const toProperCase = (value: string) =>
   value.replace(/\w\S*/g,
     (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase())
+
+export const isRemoteImageURL = (url?: string) =>
```

Comment:
Can we add a test for this util? 🙏

Reply:
https://github.com/brave/brave-core/pull/11518/commits/cbb07ff5299e7a022398272cc5ff9c3fd277bb41

Reply:
Thanks! Looks good!